src/views/Claims.vue

```diff
@@ -48,27 +63,32 @@ export default class Claims extends Vue {
     contract!: Contract;
     reward!: number;
     tokenAndAmount: Token[] = [];
-    error = false;
+    error = '';
+    walletExist = false;
+    loading = false;
     /**
      * Connects user to metamask, after that the reward variable is updated
      */
     async connect() {
         // set address of smart-contract, found in modules-solidity after command npx hardhat node
         await this.$store.dispatch('metamask/connect');
         this.contract = new this.web3.eth.Contract(feeCollectorAbi as AbiItem[], FEE_COLLECTOR_ADDRESS);
-        // when connected trough metamask update reward variable
+        // when connected trough metamask update reward variable;
+        const response = await axios.get(API_URL_GET_WALLET + this.account);
```

Comment:
It's better to leave the section that is mostly static in. The reason for this is that this is position specific as well.
The URL prefix isn't, that is always the same for the API. Could do something like: ```${VUE_APP_API_URL}\wallet\${this.account}```

Storing all possible routes in the .env. creates a lot of clutter.